eZ/Bundle/EzPublishCoreBundle/Resources/config/templating.yml

```diff
@@ -302,6 +302,9 @@ services:
             - $content.default_ttl$
         tags:
             - { name: kernel.event_subscriber }
+        calls:
+            - [setConfigurator, ['@ezplatform.http_cache.response_configurator']]
+            - [setDispatcherTagger, ['@ezplatform.http_cache.response_tagger.dispatcher']]
```

Comment:
If you want to go the BC way, assuming that this listener is extended by some:

* Create a new service with all you need and make it inherit from the deprecated one
* On the deprecated one, you'll probably need to change visibility on properties
* Trigger deprecation errors from the deprecated one.
* Redefine the service to match the new class

WDYT?